ui/Playlists.php

```diff
@@ -1182,9 +1203,14 @@ private function emitTrackField($tag, $seltrack, $id) {
     private function emitEditForm($playlistId, $id, $album, $track) {
       $entry = new PlaylistEntry($album);
       $playlist = Engine::api(IPlaylist::class)->getPlaylist($playlistId, 1);
-      $showTimeAr = $this->zkTimeRangeToISOTimeAr($playlist['showtime']);
+      $showTimeAr = $this->zkTimeRangeToISODateTimeAr($playlist);
       $startAMPM = self::timestampToAMPM($showTimeAr[0]);
-      $endAMPM = self::timestampToAMPM($showTimeAr[1]);
+      $endTime = new \DateTime($showTimeAr[1]);
+      $nowTime = new \DateTime("now");
+      $endTime = $endTime > $nowTime ? $nowTime : $endTime;
```

Comment:
This condition means it is not possible to set the time in future playlists.  Not sure how common a use case that is, but let's take this example:
1. Playlist from 1400-1500.
2. Current time is 1102
3. When I edit an entry in this playlist, the time displayed in parenthesis is:
    (2:00pm - 11:02 AM)

The range doesn't make sense.  Also note the AM/PM suffixes are different cases.